src/citrine/informatics/predictors.py

```diff
@@ -218,6 +224,7 @@ def stuff_predictor_into_envelope(predictor: dict) -> dict:
         return dict(
             module_type='PREDICTOR',
             config=predictor,
+            # TODO: What's this for?
```

Comment:
Is this question directed at the `stuff_predictor_into_envelope` method?

Reply:
Yes, but mostly I was confused why the active flag was set to False here

Reply:
It was arbitrary. The deserialization code for modules assumes they come in this envelope with the `config` field pointing to the module itself and `module_type`, `active`, and `schema_id` being the other required fields. There are also some optional fields.
Embedded predictors don't come with this envelope, so rather than re-doing the serde code I just made a dummy envelope. The info in the `active` and `schema_id` fields gets thrown away upon deserialization so I just chose arbitrary values.